src/v0/destinations/june/transform.js

```diff
@@ -48,7 +53,19 @@ const trackResponseBuilder = (message, destination) => {
   return responseBuilder(payload, endpoint, destination);
 };
 
-// ref :- https://www.june.so/docs/api#:~:text=Copy-,Identifying%20companies,-(optional)
+/**
+ * Ref : https://www.june.so/docs/tracking/http-api/page
+ * @param {*} message
+ * @param {*} destination
+ * @returns
+ */
+const pageResponseBuilder = (message, destination) => {
+  const { endpoint, name } = CONFIG_CATEGORIES.PAGE;
+  const payload = constructPayload(message, MAPPING_CONFIG[name]);
+  return responseBuilder(payload, endpoint, destination);
+};
+
+// ref :- https://www.june.so/docs/tracking/http-api/group
 const groupResponseBuilder = (message, destination) => {
```

Comment:
group, page, identify are similar so make single function for all of them

Reply:
Keeping all the calls code in separate function is more readable and also it will help us to accommodate future changes easily

Reply:
we can always refactor it if necessary, writing duplicate should be avoided as if we forget to change in one place then it will be problematic

Reply:
Refactored repetitive code